backend/src/functions/onTransaction/ledger.ts

```diff
@@ -0,0 +1,34 @@
+// TODO: Make a lambda layer for this file and model_properties
```

Comment:
Sure? I didn't need lambda layers so far, and they sound like a decent bit of extra work.